packages/datepickers/src/styled/styled-datepicker.tsx

```diff
@@ -52,36 +54,40 @@ export const StyledHeaderPaddle = styled.div<{
   display: flex;
   justify-content: center;
   align-items: center;
-  color: ${PALETTE.grey[600]};
+  color: ${props => getColor('neutralHue', 600, props.theme)};
   cursor: pointer;
   border-radius: 50%;
 
   ${props => props.isHidden && `visibility: hidden;`};
 
   :hover {
-    color: ${PALETTE.grey[800]};
-    background-color: ${rgba(PALETTE.blue[600], 0.08)};
+    color: ${props => getColor('foreground', 800, props.theme)};
```

Comment:
@jzempel let's have some sit-down time to figure out `foreground` vs `neutralHue` usages.